package.json

```diff
@@ -142,6 +143,7 @@
     "rxjs": "^7.4.0",
     "semver": "^7.3.5",
     "supports-color": "9.2.2",
+    "terser": "^5.14.1",
```

Comment:
FYI: Strictly not needed because it is transitively already pulled, but technically `@bazel/terser` has a peer dependency on it, and I wanted to update to the latest version which includes my source-map fix!